pwiz_tools/Shared/CommonUtil/SystemUtil/ProcessRunner.cs

```diff
@@ -215,6 +222,13 @@ public static bool GoodIfExitCodeIsZero(string stderr, int exitCode)
                 }
 
             }
+            catch (IOException ex)  // CONSIDER: Should we handle more types like WrapAndThrowException does?

```

Comment:
I would probably catch all exception types here, and not worry about the fact that the type of the exception might be getting changed to IOException from whatever its original type might have been.
But this way is fine too.

Reply:
Okay. That makes sense. I was thinking that catching Exception you would throw Exception, but if you catch exception and throw IOException that seems a bit better to me.